packages/ui/src/components/atoms/Spinner.ts

```diff
@@ -0,0 +1,39 @@
+import styled, { keyframes } from 'styled-components'
+import { colors } from '../../styles'
+
+const spinnerAnimation = keyframes`
+from {
+        transform: rotate(0deg);
+    }
+    to {
+        transform: rotate(360deg);
+    }
+`
+
+const sizes = {
+  xsmall: '18px',
+  small: '50px',
+  medium: '70px',
+  big: '133px',
+}
+
+type TSpinner = {
+  size?: 'xsmall' | 'small' | 'medium' | 'big'
+}
+
+const Spinner = styled.div.attrs<TSpinner>((props) => ({
+  size: props?.size,
+}))<TSpinner>`
```

Comment:
Is there any console warning with the prop?

Reply:
Nothing about props.